### PR TITLE
[Tyra1] Version selector for tyra future releases including tyrav2

### PR DESCRIPTION
**Pull request by Wolf3s**

### CODE REVIEW

File: src/engine/Makefile [hunk 1]
```diff
@@ -24,6 +24,7 @@ EE_OBJS =								\
 	modules/vif_sender.o				\
 	utils/math.o						\
 	utils/string.o						\
+	utils/Tyra_version.o				\
```
Comment (h4570) [inline src/engine/Makefile]: Please do `tyra_version.o`.

Pretty cool idea, overall!

Comment (h4570) [reply src/engine/Makefile]: When it comes to tyrav2.
I think that the good place will be `/info/version.hpp` and we can include this class in `/info.hpp`

Comment (Wolf3s) [reply src/engine/Makefile]: > When it comes to tyrav2. I think that the good place will be `/info/version.hpp` and we can include this class in `/info.hpp`

Yes it sounds good

**Comment by h4570** [comment]
Included in Tyrav2.
I marked you as author of this feature.

**Pull request closed without merging**